src/authentication/register.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require( 'jsonwebtoken' )
+const crypto = require( 'crypto' )
+const User = require( '../../models/users' )
+const { generateToken, userInfo, tokenInfo } = require( './authentication' )
+
+const ERROR_MESSAGE = 'Cannot authenticate request'
+
+exports.register = (request, response, next) => {
+  const { email, password } = request.body
+  console.log('email', email )
+  console.log('body', request.body )
+
+  if (!email) {
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```